tests/framework/e2e/cluster.go

```diff
@@ -17,6 +17,7 @@ package e2e
 import (
 	"context"
 	"fmt"
+	"go.etcd.io/etcd/tests/v3/framework/config"
```

Comment:
Please put all "go.etcd.io/..." items together

Reply:
Would it make sense to enable smth like https://golangci-lint.run/usage/install/#ci-installation for this repo?